config-sample-ent-drive.yaml

```diff
@@ -20,6 +20,9 @@ test_config:
     fio_runtime: 1200
     fio_ramptime: 60
     max_ns: 32 # Max number of namespaces for the drive
+    # If specified, this option will override the IO engine used for tests from libaio to specified engine
+    # Can be an IO engine supported by OS, for ex: psync/sync/io_uring/windowsaio etc.
+    # io_engine: io_uring
```

Comment:
Given the default is libaio, I'd probably just uncomment this and leave it as

```
io_engine: libaio
```

Reply:
thanks, made the change